server/app.js

```diff
@@ -18,7 +18,7 @@ app.use(bodyParser.urlencoded({extended: false}))
 app.use(bodyParser.json())
 
 // routes
-app.use('/user', userRoutes)
+app.use('/', indexRoutes)
```

Comment:
missing semicolon? Also on line 18

Reply:
Why at some places there is semi colon and some places there is not? Let's follow one style throughout.

Reply:
need a linter yes , ill make a pull for that?

Reply:
We do have one. Just not before you submitted the PR. Can you pull the latest changes from master and then see if it highlights the issues?